crates/wasmi/src/engine/code_map.rs

```diff
@@ -244,32 +254,6 @@ impl CodeMap {
         func.init_compiled(entity);
     }
 
-    /// Initializes the [`EngineFunc`] for lazy translation.
-    ///
-    /// # Panics
-    ///
-    /// - If `func` is an invalid [`EngineFunc`] reference for this [`CodeMap`].
-    /// - If `func` refers to an already initialized [`EngineFunc`].
-    pub fn init_func_as_uncompiled(
-        &self,
-        func: EngineFunc,
-        func_idx: FuncIdx,
-        bytes: &[u8],
-        module: &ModuleHeader,
-        func_to_validate: Option<FuncToValidate<ValidatorResources>>,
-    ) {
-        let mut funcs = self.funcs.lock();
-        let Some(func) = funcs.get_mut(func) else {
-            panic!("encountered invalid internal function: {func:?}")
-        };
-        func.init_uncompiled(UncompiledFuncEntity::new(
-            func_idx,
-            bytes,
-            module.clone(),
-            func_to_validate,
-        ));
-    }
-
```

Comment:
Please don't unnecessarily move functions around that have no effect on the PRs diff. This just makes it harder to review and blow-up the PR's diff.